docs/system-admin-guide/authentication/saml/README.md

```diff
@@ -320,27 +328,35 @@ Your identity provider may optionally encrypt the assertion response, however no
 To configure assertion encryption, you need to provide the certificate to send in the request and private key to decrypt the response:
 
 ```yaml
-  certificate: "-----BEGIN CERTIFICATE-----\n .... certificate contents ....\n-----END CERTIFICATE-----",
-  private_key: "-----BEGIN PRIVATE KEY-----\n .... private key contents ....\n-----END PRIVATE KEY-----"
+default:
+  # <-- other configuration -->
+    mysaml1:
+      # <-- other configuration -->
+      certificate: "-----BEGIN CERTIFICATE-----\n .... certificate contents ....\n-----END CERTIFICATE-----"
+      private_key: "-----BEGIN PRIVATE KEY-----\n .... private key contents ....\n-----END PRIVATE KEY-----"
 ```
 
 Request signing means that the service provider (OpenProject in this case) uses the certificate specified to sign the request to the identity provider. They reuse the same `certificate` and `private_key` settings as for assertion encryption. It is recommended to use an RSA key pair, the key must be provided without password.
 
 To enable request signing, enable the following flag:
 
 ```yaml
-  certificate: "-----BEGIN CERTIFICATE-----\n .... certificate contents ....\n-----END CERTIFICATE-----",
-  private_key: "-----BEGIN PRIVATE KEY-----\n .... private key contents ....\n-----END PRIVATE KEY-----",
-  security: {
-    # Whether SP and idP should sign requests and assertions
-    authn_requests_signed: true,
-    want_assertions_signed: true,
-    # Whether the idP should encrypt assertions
-    want_assertions_signed: false,
-    embed_sign: true,
-    signature_method: 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256',
-    digest_method: 'http://www.w3.org/2001/04/xmlenc#sha256',
-  }
+default:
+  # <-- other configuration -->
+    mysaml1:
+      # <-- other configuration -->
+      certificate: "-----BEGIN CERTIFICATE-----\n .... certificate contents ....\n-----END CERTIFICATE-----"
+      private_key: "-----BEGIN PRIVATE KEY-----\n .... private key contents ....\n-----END PRIVATE KEY-----"
+      security: {
+        # Whether SP and idP should sign requests and assertions
+        authn_requests_signed: true,
+        want_assertions_signed: true,
+        # Whether the idP should encrypt assertions
+        want_assertions_signed: false,
+        embed_sign: true,
+        signature_method: 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256',
+        digest_method: 'http://www.w3.org/2001/04/xmlenc#sha256',
+      }
```

Comment:
```suggestion
      security:
        # Whether SP and idP should sign requests and assertions
        authn_requests_signed: true
        want_assertions_signed: true
        # Whether the idP should encrypt assertions
        want_assertions_signed: false
        embed_sign: true
        signature_method: 'http://www.w3.org/2001/04/xmldsig-more#rsa-sha256'
        digest_method: 'http://www.w3.org/2001/04/xmlenc#sha256'
```

Hi @adam-op , there should be no braces and commas in this block

Reply:
Thank you Oliver for your fast help
I commited the last agreed changes, 
I think we have improved the SAML docs a lot (again)
and i hope that this is the last change before the GUI will be developed and working.